### PR TITLE
Javascript Library v1.1

### PR DESCRIPTION
**Pull request by UmairAhmad125**
In this project, a user can
- Add book
- Delete book
- change read status of book

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  const book = new Book(name, author, pages,status);
+  myBooks.push(book);
+}
+
+function displaybook(){
+bookcontainer.innerHTML="";
+
+myBooks.forEach((item,index) => {
+const card=document.createElement("div");
+console.log(card);
```
Comment (whiz25) [inline js/main.js:41]: - remove that console.log

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+card.className="col-6"
+card.innerHTML=`<div class="card text-dark bg-white mb-3" >
+  <li class="list-group-item d-flex justify-content-between"><p class="bg-dark px-2 py-1 text-white">${index+1}</p><a href="#" class="delete text-danger" onclick="removeBook(${index})" >X</a></li>
+  <li class="list-group-item "> Book Name: ${item.name}</li>
+  <li class="list-group-item ">Author Name: ${item.author}</li>
+  <li class="list-group-item ">Number of pages: ${item.pages}</li>
+  <li class="list-group-item "><a href="#" class="btn-warning px-2 py-1 rounded status">${item.status}</a></li>
+  </div>`
```
Comment (whiz25) [inline js/main.js:49]: - don't mix js logic with DOM logic. This piece of code should live in a separate file that is responsible for rendering books in the DOM. Create a new file for that.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
You have done well in this project. App looks good and works well. Let's make some changes before we can merge this PR.
- [x] fix all linter errors
- [x] add a title to this PR.
- [x] validate the fields for creating a new book. right now one can create a new book empty fields. this shouldn't happen.
### Optional
- [ ] separate your business logic from DOM logic. any code that renders HTML should be in a file that renders DOM. Your main.js file is cluttered with business and DOM logic.


### Martin Afani

**Comment by ajkacca457** [comment]
@whiz25  Thanks for your critical review. Your review helped us in case of improving our project. We have added proper validation to the form. Now users can not create books with empty filed input.  We have also replaced the PR title and fixed linter errors. Hopefully this time our project will be acceptable.

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @UmairAhmad125 and @ajkacca457, congratulations on the hard work on this project :fireworks: Having fixed all the comments from the previous TSE :smile: I hereby approve the project :+1: 

Go ahead and merge into development :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**